scripts/git-pre-push.sh

```
@@ -8,6 +8,12 @@ if ! make lint; then
     exit 1
 fi
 
+echo "\n=== Running vulnerability check ==="
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)